config/chance_weight.cwt

```diff
@@ -1,53 +1,35 @@
 enums = {
+    # Aren't these just value fields?
```

Comment:
Possible, but since they can go on the LHS, I'm not sure using value_field instead would be appropriate.